src/test/java/hudson/security/PAMSecurityRealmTest.java

```diff
@@ -12,9 +16,14 @@
 /**
  * @author Kohsuke Kawaguchi
  */
-public class PAMSecurityRealmTest extends HudsonTestCase {
-    public void testLoadUsers() {
-        if (Functions.isWindows())  return; // skip on Windows
+public class PAMSecurityRealmTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    @Test
+    public void loadUsers() {
+        if (Functions.isWindows()) return; // skip on Windows
```

Comment:
You can use `assumeTrue(Functions.isWindows());`

Reply:
I guess you meant `assumeFalse(...)`. I just applied this change.